lib/guardian/db/sweeper.ex

```diff
@@ -0,0 +1,70 @@
+defmodule Guardian.DB.Sweeper do
```

Comment:
For backwards compatibility we could add a `Guardian.DB.Token.SweeperServer` alias but I'd like to encourage folks to move away from that module.

Reply:
@doomspork wouldn't be better to use `SweeperServer` following some `GenServer` lingos?

It is a stateful thing (hint the server part of it) that you send messages to it 🤔 💭

Reply:
I don't often seen things called "Server" in Elixir/Erlang space, do you? Looking at a few of my applications there's nothing in the application tree with that suffix :thinking:

Reply:
It depends if you have the `Server` being the message passing, while the thing without `Server` (as it was before), was the implementation of the thing.

Dave has an amazing course about it, https://codestool.coding-gnome.com/courses/elixir-for-programmers

Reply:
I appreciate Dave's pattern but as much as he would like to see this catch on I can't say I see this having gained any adoption. I prefer to keep code simple and easy to grok for the human reader over trying to be "academically right".

I'd like to leave this.